internal/run/convert_test.go

```diff
@@ -0,0 +1,54 @@
+package run
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/pelotech/drone-helm3/internal/env"
+)
+
+type ConvertTestSuite struct {
+	suite.Suite
+	ctrl            *gomock.Controller
+	mockCmd         *Mockcmd
+	originalCommand func(string, ...string) cmd
+}
+
+func (suite *ConvertTestSuite) BeforeTest(_, _ string) {
+	suite.ctrl = gomock.NewController(suite.T())
+	suite.mockCmd = NewMockcmd(suite.ctrl)
+
+	suite.originalCommand = command
+	command = func(path string, args ...string) cmd { return suite.mockCmd }
+}
+
+func (suite *ConvertTestSuite) AfterTest(_, _ string) {
+	command = suite.originalCommand
+}
+
+func TestConvertTestSuite(t *testing.T) {
+	suite.Run(t, new(ConvertTestSuite))
+}
+
+func (suite *ConvertTestSuite) TestNewConvert() {
+	cfg := env.Config{
```

Comment:
This should probably have tests to cover the required flags and the logic in Execute outside of the .Run().

Reply:
Yes, test cases especially for failure conditions should be covered.